go.mod

```diff
@@ -7,11 +7,11 @@ require (
 	github.com/onsi/ginkgo v1.14.1
 	github.com/onsi/gomega v1.10.1
 	github.com/pkg/errors v0.9.1
-	github.com/projectcalico/api v0.0.0-20210706154750-8686d6f77130
+	github.com/projectcalico/api v0.0.0-20210713142641-eb76b6b20ac1
 	github.com/projectcalico/cni-plugin v1.11.1-0.20210713192619-3dcf4cdb4636
 	github.com/projectcalico/felix v0.0.0-20210705102911-d04ee2c303fd
-	github.com/projectcalico/libcalico-go v1.7.2-0.20210712155928-719c24d19752
-	github.com/projectcalico/typha v0.7.3-0.20210706170700-733c771df430
+	github.com/projectcalico/libcalico-go v1.7.2-0.20210714231320-ee4edd4a972f
```

Comment:
I would expect this to match https://github.com/projectcalico/libcalico-go/commit/6200900018095240931e4efe7fe967b1d5e40529

Reply:
Disregard the previous, thought this was against `master`. I do think the pin might need to be updated to match https://github.com/projectcalico/libcalico-go/commit/f507c0f847c3aac947bd066da875376fd7290ce4 though